src/storage/resourceManager.ts

```diff
@@ -412,19 +412,19 @@ export class ResourceManager {
    * @param subjects
    */
   async setSubjects(
-    schemaRegistry: SchemaRegistry,
+    schemaRegistryKeyable: ISchemaRegistryResource,
```

Comment:
Renamed the param as well as re-typed it to the new interface. The renaming caused the following churn.